frontend_v2/src/app/components/home/home.component.scss

```diff
@@ -5,6 +5,10 @@
   margin-right: auto;
 }
 
+.twitter-container {
+  margin-left:15vw;
```

Comment:
using `vw` will make it misbehave for different browsers. Can we please use px instead?

Reply:
Changed!